.github/workflows/test.yml

```diff
@@ -24,13 +24,13 @@ jobs:
             exit 0
           fi
 
-          npm ci --production --only=prod
```

Comment:
npm ci --production --only=prod will avoid installing devDependencies, which mimicks the environment on production better?